teuthology/orchestra/daemon/cephadmunit.py

```diff
@@ -112,6 +112,9 @@ def running(self):
         """
         return self.is_started
 
+    def finished(self):
+        return self.is_started
```

Comment:
should this not use the `self.status_cmd` to determine if the container is still running?

Reply:
Made the change, thanks!